src/main/kotlin/no/nav/medlemskap/regler/v1/ReglerForGrunnforordningen.kt

```diff
@@ -11,8 +11,8 @@ class ReglerForGrunnforordningen(val personfakta: Personfakta) : Regler() {
             }
 
     private val erBrukerEØSborger = Regel(
-            identifikator = "GRUNNFORORDNING-EØS",
-            avklaring = "Er brukeren statsborger i et EØS land?",
+            identifikator = "2",
+            avklaring = "Er bruker omfattet av grunnforordningen (EØS)? Dvs er bruker statsborger i et EØS-land inkl. Norge?",
```

Comment:
Denne begynner kanskje å bli i lengste laget for Grafana..?

Reply:
Jeg er også litt usikker på om Grafana er så glad i for mange spesialtegn.. Det er derfor jeg måtte legge til de `replace("?", "")` koden. Så egentlig bør vi unngå spørsmålstegn, og kanskje også andre ting, som paranteser, bindestrek og punktum. (Dette vet jeg ikke, jeg bare gjetter!)

Reply:
Men fjernes det ikke uansett? Eller skal jeg fjerne alle spørsmålstegn?

Reply:
Men fjernes det ikke uansett med replace(..)? eller skal jeg fjerne alle spørsmålstegn i koden?

Reply:
Jeg kortet ned spørsmålet, trur jeg kan merge da om du ikke har noe mer